python/ml4ir/applications/ranking/tests/config/auto_dag_network_sample_config.yaml

```diff
@@ -0,0 +1,38 @@
+architecture_key: auto-dag-network
+layers:
+  - type: keras.layers.merge.Concatenate
+    name: features_concat
+    inputs:
+      - query_text
+      - text_match_score
+      - page_views_score
+    aslist: true
+    axis: -1
```

Comment:
@arvindsrikantan Worth putting the keras layers args inside a nested struct here. Maybe `layer_kwargs`?

Reply:
I'll stick to `args` instead of `layer_kwargs` to be consistent with feature layer.